### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

I updated the README.md file following #121 issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,23 +24,30 @@ On release, automated continuous integration tests run the pipeline on a full-si
 
 1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
 2. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
-3. Quality trimming and adapters removal for raw reads ( [`Trimm Galore!`](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/))
-4. Merge trimmed, pair-end reads ( [`Seqtk`](https://github.com/lh3/seqtk))
-5. Choice of de novo assembly:
+3. Quality trimming and adapters removal for raw reads [`Trimm Galore!`](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/)
+4. Filter sequences with [`BBduk`](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/)
+5. Normalize the sequences with KHMER or [`BBnorm`](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/)
```
Comment (erikrikarddaniel) [inline README.md]:
```suggestion
5. Normalize the sequences with [khmer](https://khmer-protocols.readthedocs.io/en/latest/mrnaseq/2-diginorm.html) or [`BBnorm`](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/)
```

File: README.md [hunk 2]
```diff
@@ -24,23 +24,30 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
+6. Merge trimmed, pair-end reads ( [`Seqtk`](https://github.com/lh3/seqtk))
+7. Choice of de novo assembly:
```
Comment (erikrikarddaniel) [inline README.md]:
```suggestion
7. Choice of de novo assembly programs:
```

File: README.md [hunk 3]
```diff
@@ -24,23 +24,30 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
    2. Mapping cleaned reads to the assembly for quantification [`BBmap`](https://sourceforge.net/projects/bbmap/)
    3. Get raw counts per each gene present in the assembly [`Featurecounts`](http://subread.sourceforge.net) -> TSV table with collected featurecounts output
-8. Taxonomical annotation:
-   1. [`EUKulele`](https://github.com/AlexanderLabWHOI/EUKulele) -> Reformat TSV output "Reformat_tax.R"
-9. Choice of functional annotation: 1. [`Eggnog-mapper`](http://eggnog-mapper.embl.de) 2. [`Run-DBcan`](https://github.com/linnabrown/run_dbcan) 3. [`Hmmsearch`](https://www.ebi.ac.uk/Tools/hmmer/search/hmmsearch). Besides searching the ORFs, each ORF's hits will be ranked.
-   10 Summary statistics table. Collect_stats.R
+10. Functional annotation:
+11. [`Eggnog`](https://github.com/eggnogdb/eggnog-mapper) -> Reformat TSV output "eggnog table"
+12. [`KOfamscan`](https://github.com/takaram/kofam_scan)
+13. [`HMMERsearch`](https://www.ebi.ac.uk/Tools/hmmer/search/hmmsearch) -> Ranking orfs based on HMMprofile with [`Hmmrank`](https://github.com/erikrikarddaniel/hmmrank)
```
Comment (erikrikarddaniel) [inline README.md]:
```suggestion
    1. [`Eggnog`](https://github.com/eggnogdb/eggnog-mapper) -> Reformat TSV output "eggnog table"
    2. [`KOfamscan`](https://github.com/takaram/kofam_scan)
    3. [`HMMERsearch`](https://www.ebi.ac.uk/Tools/hmmer/search/hmmsearch) -> Ranking orfs based on HMMprofile with [`Hmmrank`](https://github.com/erikrikarddaniel/hmmrank)
```

File: README.md [hunk 4]
```diff
@@ -24,23 +24,30 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
+14. Taxonomical annotation:
+15. [`EUKulele`](https://github.com/AlexanderLabWHOI/EUKulele) -> Reformat TSV output "Reformat_tax.R"
+16. [`CAT`](https://github.com/dutilh/CAT)
+17. Choice of functional annotation: 1. [`Eggnog-mapper`](http://eggnog-mapper.embl.de) 2. [`Run-DBcan`](https://github.com/linnabrown/run_dbcan) 3. [`Hmmsearch`](https://www.ebi.ac.uk/Tools/hmmer/search/hmmsearch). Besides searching the ORFs, each ORF's hits will be ranked.
```
Comment (erikrikarddaniel) [inline README.md]:
```suggestion
    1. [`EUKulele`](https://github.com/AlexanderLabWHOI/EUKulele) -> Reformat TSV output "Reformat_tax.R"
    2. [`CAT`](https://github.com/dutilh/CAT)
    3. Choice of functional annotation: 1. [`Eggnog-mapper`](http://eggnog-mapper.embl.de) 2. [`Run-DBcan`](https://github.com/linnabrown/run_dbcan) 3. [`Hmmsearch`](https://www.ebi.ac.uk/Tools/hmmer/search/hmmsearch). Besides searching the ORFs, each ORF's hits will be ranked.
```

**Review by erikrikarddaniel — APPROVED** [review]
A few small phrasing things, otherwise fine.

File: README.md [hunk 5]
```diff
@@ -43,11 +43,11 @@ On release, automated continuous integration tests run the pipeline on a full-si
     1. [`Eggnog`](https://github.com/eggnogdb/eggnog-mapper) -> Reformat TSV output "eggnog table"
     2. [`KOfamscan`](https://github.com/takaram/kofam_scan)
     3. [`HMMERsearch`](https://www.ebi.ac.uk/Tools/hmmer/search/hmmsearch) -> Ranking orfs based on HMMprofile with [`Hmmrank`](https://github.com/erikrikarddaniel/hmmrank)
-14. Taxonomical annotation:
+11. Taxonomical annotation:
     1. [`EUKulele`](https://github.com/AlexanderLabWHOI/EUKulele) -> Reformat TSV output "Reformat_tax.R"
     2. [`CAT`](https://github.com/dutilh/CAT)
     3. Choice of functional annotation: 1. [`Eggnog-mapper`](http://eggnog-mapper.embl.de) 2. [`Run-DBcan`](https://github.com/linnabrown/run_dbcan) 3. [`Hmmsearch`](https://www.ebi.ac.uk/Tools/hmmer/search/hmmsearch). Besides searching the ORFs, each ORF's hits will be ranked.
-    10 Summary statistics table. Collect_stats.R
+       10 Summary statistics table. Collect_stats.R
```
Comment (erikrikarddaniel) [inline README.md]: This is not correct, right? First, there should be a `.` behind the number, it's not number 10 and should it really be indented?

**Review by erikrikarddaniel — COMMENTED** [review]
One more.

File: README.md (hunk 5 above)
Comment (danilodileo) [reply README.md]: yes, this should be number 12

Comment (erikrikarddaniel) [reply README.md]: And the indentation, is that correct?

**Pull request merged**